src/content/1.7/modules/testing/ci-cd.md

```diff
@@ -82,7 +82,7 @@ jobs:
       # Run a container for PHPStan, having access to the module content and PrestaShop sources.
       # This tool is outside the composer.json because of the compatibility with PHP 5.6
       - name : Run PHPStan
-        run: docker run --rm --volumes-from temp-ps -v $PWD:/web/module -e _PS_ROOT_DIR_=/var/www/html --workdir=/web/module phpstan/phpstan:0.11.19 analyse --configuration=/web/module/tests/phpstan/phpstan.neon
+        run: docker run --rm --volumes-from temp-ps -v $PWD:/web/module -e _PS_ROOT_DIR_=/var/www/html --workdir=/web/module phpstan/phpstan:0.11.19 analyse --configuration=/web/module/tests/phpstan/phpstan.neon .
```

Comment:
```suggestion
        run: docker run --rm --volumes-from temp-ps -v $PWD:/web/module -e _PS_ROOT_DIR_=/var/www/html --workdir=/web/module phpstan/phpstan:0.11.19 analyse --configuration=/web/module/tests/phpstan/phpstan.neon ./
```

Reply:
There is no reason to add the / since `.` means "current directory"